packages/menu-bar/src/vaadin-menu-bar-mixin.js

```diff
@@ -158,7 +167,7 @@ export const MenuBarMixin = (superClass) =>
 
     static get observers() {
       return [
-        '_themeChanged(_theme, _overflow, _container)',
+        '_themeChanged(_theme, _overflow, _container, reverseCollapse)',
```

Comment:
This part was not immediately obvious, as in how are all these things related to the theme changing. Is there a better name for this observer? Or should we add a separate observer for `reverseCollapse`?

Reply:
I'd rather rename it because it's already used for dependencies other than just theme

Reply:
After giving it more though, a separate observer makes more sense after all. Done.